script.js

```diff
@@ -1,16 +1,105 @@
 // Write your JavaScript code here!
+window.addEventListener("load", function(){
+    let listedPlanets;
+    // Set listedPlanetsResponse equal to the value returned by calling myFetch()
+    listedPlanets = "https://handlers.education.launchcode.org/static/planets.json";
 
-window.addEventListener("load", function() {
-
-   let listedPlanets;
-   // Set listedPlanetsResponse equal to the value returned by calling myFetch()
-   let listedPlanetsResponse;
-   listedPlanetsResponse.then(function (result) {
-       listedPlanets = result;
-       console.log(listedPlanets);
-   }).then(function () {
-       console.log(listedPlanets);
-       // Below this comment call the appropriate helper functions to pick a planet fom the list of planets and add that information to your destination.
-   })
-   
-});
\ No newline at end of file
+    // Fetching Planetary Data
+    this.fetch(listedPlanets).then(function(result) {
+        //listedPlanets=results;
+        result.json().then(function(json) {
+            let listedPlanetsResponse = document.getElementById("missionTarget");
+            //console.log(listedPlanets);
+            let missionDestination = Math.floor(Math.random()*json.length -1);
+           listedPlanetsResponse.innerHTML = `
+           <h1>Mission Destination</h1>
+           <ol>
+              <li>Name: ${json[missionDestination].name}</li>
+              <li>Diameter: ${json[missionDestination].diameter}</li>
+              <li>Star: ${json[missionDestination].star}</li>
+              <li>Distance from Earth: ${json[missionDestination].distance}</li>
+              <li>Number of Moons: ${json[missionDestination].moons}</li>
+           </ol>
+           <img src="${json[missionDestination].image}"> `;
+        });
+     });
+
+
+//Submitting pilot, co-pilot, fuel level, & cargo mass
+let form = document.querySelector("form");
+form.addEventListener("submit", function(event){
+    let pilotName = document.querySelector("input[name=pilotName]");
+    let copilotName = document.querySelector("input[name=copilotName]");
+    let fuelLevel = document.querySelector("input[name=fuelLevel]");
+    let cargoMass = document.querySelector("input[name=cargoMass]");
+
+    let faultyItems = document.getElementById('faultyItems');
+    let fuelStatus = document.getElementById('fuelStatus');
+    let launchStatus = document.getElementById('launchStatus');
+    let pilotStatus = document.getElementById('pilotStatus');
+    let copilotStatus = document.getElementById('copilotStatus');
```

Comment:
```suggestion
formSubmission(
      document,
      faultyItems,
      pilotName.value,
      copilotName.value,
      fuelLevel.value,
      cargoMass.value
    );
```
Instead of grabbing the status elements here, add the call to your `formSubmission()` function  and pass in its associated arguments. You can use your status elements in your `formSubmission()` in scriptHelper instead to update the list.

Reply:
Once you move lines 37-40 to `scriptHelper.js`, you'll also need a variable to store the cargo status: `let cargoStatus = document.getElementById('cargoStatus')`.